src/app/static/src/tests/testHelpers.ts

```diff
@@ -35,3 +38,23 @@ export function testUploadErrorCommitted(url: string,
         });
     });
 }
+
+export function expectAllMutationsDefined(mutationDefinitions: any, mutationTree: MutationTree<any>) {
```

Comment:
Consolidated the test helper classes into one file, this used to live in it's own file.